data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -943,6 +944,7 @@
     "type": "GENERIC",
     "copy-from": "longsword",
```

Comment:
`"copy-from": "longsword"`

The Mage blade should have categories since it copies from Longsword. The Mage Blade is a valid Medieval Swordsmanship weapon at the moment.

Perhaps Stormhammer should copy from Warhammer instead of having categories added?